docs/ref/files/storage.txt

```diff
@@ -85,6 +85,11 @@ The ``Storage`` class
         able to return the last accessed time this will raise
         ``NotImplementedError`` instead.
 
+        .. deprecated:: 1.10
+
+           You should use :meth:`get_accessed_time()
```

Comment:
more simply:
```
:meth:`~django.core.files.storage.Storage.get_accessed_time`
```
(that'll add the parentheses in the rendered version too)
Actually, since you're in the `Storage` class already,
```
:meth:`get_accessed_time`
```
is probably enough.